Fatras/include/ActsFatras/Physics/PhotonConversion/PhotonConversion.hpp

```diff
@@ -0,0 +1,311 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2021 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Definitions/Units.hpp"
+#include "Acts/Material/MaterialSlab.hpp"
+#include "Acts/Utilities/UnitVectors.hpp"
+#include "ActsFatras/EventData/Particle.hpp"
+#include "ActsFatras/EventData/ProcessType.hpp"
+#include "ActsFatras/Utilities/ParticleData.hpp"
+
+#include <limits>
+#include <random>
+#include <vector>
+
+#include <math.h>
+
+namespace ActsFatras {
+/// This class handles the photon conversion. It evaluates the distance
+/// after which the interaction will occur and the final state due the
+/// interaction itself.
+class PhotonConversion {
+ public:
+  using Scalar = ActsFatras::Particle::Scalar;
+  /// Lower energy threshold for produced children
+  Scalar minimumMomentum = 100. * Acts::UnitConstants::MeV;
+  /// Scaling factor of children energy
+  Scalar childEnergyScaleFactor = 2.;
+  /// Scaling factor for photon conversion probability
+  Scalar conversionProbScaleFactor = 0.98;
```

Comment:
Just for clarification: are these tuning parameters to optimize matching to the full simulation or are these non-physical scale factors for testing?

Reply:
These are tuning parameters, set as in the ISF